pkg/arvo/ted/ph/moon-az.hoon

```diff
@@ -12,6 +12,7 @@
 ;<  ~  bind:m  (init-ship ~bud |)
 ;<  ~  bind:m  (init-ship ~marbud |)
 ;<  ~  bind:m  (init-ship ~linnup-torsyx |)
+::TODO  not supported!
```

Comment:
What's not supported, moons?

Reply:
That's right. Went ahead and added (the dumbest possible) moon support. Seems to work, except that moons can't talk to non-sponsor galaxies.